.github/workflows/test_web.yml

```diff
@@ -82,3 +82,21 @@ jobs:
       - name: Run tests
         working-directory: web
         run: npm test
+
+  check-required:
+    needs: changes
+    if: ${{ needs.changes.outputs.src == 'false' }}
```

Comment:
You can omit the enclosing`${{ ]}`.

Reply:
It was copied from the `build` job, which includes it. I wanted to keep them the same, and I didn't want to put further refactoring into this commit to keep it targeted.